public/app/features/explore/Explore.tsx

```diff
@@ -416,8 +416,13 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
     );
   };
 
-  onClickTableCell = (columnKey: string, rowValue: string) => {
-    this.onModifyQueries({ type: 'ADD_FILTER', key: columnKey, value: rowValue });
+  onClickTableCell = (columnKey: string, rowValue: string, attributes: {}) => {
+    this.onModifyQueries({
+      key: columnKey,
+      operator: attributes['data-operator'],
```

Comment:
Explore should be agnostic about what it means to add a filter. Hence I would pass the operator on in an options map. Then it's for the datasource to decode what it needs from the options.